src/sys/socket/sockopt.rs

```diff
@@ -1001,6 +1001,133 @@ sockopt_impl!(
     bool
 );
 
+// Documented on `man 7 netlink`
+#[cfg(any(target_os = "android", target_os = "linux"))]
+sockopt_impl!(
+    /// Add socket to group
+    NetlinkAddMembership,
+    SetOnly,
+    libc::SOL_NETLINK,
+    libc::NETLINK_ADD_MEMBERSHIP,
+    libc::c_uint
```

Comment:
The man page says that this should be a signed int.  Any reason why you made it unsigned here?  Ditto for NETLINK_DROP_MEMBERSHIP.

Reply:
It will be interpreted as an unsigned int since it's a bitfield.
https://elixir.bootlin.com/linux/latest/source/net/netlink/af_netlink.c#L1641
https://elixir.bootlin.com/linux/latest/source/net/netlink/af_netlink.c#L1620